.gitignore

```diff
@@ -0,0 +1,6 @@
+# composer related
+/composer.lock
+/composer.phar
```

Comment:
This has to go into the global gitignore, not repository-local.

Reply:
I took this by example from another pear repository and thought some maintainers may have an easier live with this. But I also dont think this has to be part of the repository

Reply:
Yeah, I think this was a mistake on their part. And maybe several years ago it was OKish.

Reply:
Ah, it seems GitHub adds composer.phar by default even these days. E.g. try creating a new repository and select .gitignore template. Bummer!

I'd leave only:
```
/composer.lock
/vendor
```